### DISCUSSION TITLE
Can Edge render markdown?

### DISCUSSION THREAD

**Discussion by MichaelBelgium** [adonisjs/core#4281] [Help]
Hello!

In the first alinea of the [Edge docs](https://docs.adonisjs.com/guides/views/introduction) it says this:

> It can render any text-based format, whether is HTML, Markdown or plain text files.

But If i create `markdown.edge` and set this as content:

```
# Test

* bullet
* list
```

It doesnt seem to work? There's also no mention whatsoever about markdown

**Comment by thetutlage** [comment]
What do you mean by it does not work?

What is the expected output and what do you get?

**Reply by MichaelBelgium** [reply]
`#` should be a heading and `*` should be a list no?

But I get litteraly the raw thing, aka the same what i wrote in .edge

**Reply by Julien-R44** [reply]
There is a misunderstanding here. Edge is not a markdown to html converter

You will have to manage this part yourself. On the other hand, before passing your Markdown to your HTML converter, you can use Edge to dynamically generate certain parts of your Markdown document. That means, you can add some `@if`, `@each`, and `@include` tags in your `.md` file

Then your converter will transform it into html

**Reply by McSneaky** [reply]
Aah, from doc I always thought too that Edge has built-in markdown converter 😅

**Reply by MichaelBelgium** [reply]
Okay, i assumed "rendering" would do that 😅 so it's just wrong wording in the docs. It does nothing with markdown

**Reply by thetutlage** [reply]
<img width="719" alt="Screen Shot 2023-11-15 at 08 33 22" src="https://github.com/adonisjs/core/assets/1706381/9953687a-3ad6-497c-b8d5-5d805eca1018">

Agree, the wording on the AdonisJS docs is misleading. However, with the standalone docs of Edge we are little bit more explicit.
